packages/core/src/util/renderer.ts

```diff
@@ -1012,6 +1018,7 @@ export const mapStateToOneOfProps = (
 export interface StatePropsOfArrayLayout extends StatePropsOfControlWithDetail {
   data: number;
   minItems?: number;
+  translations?: ArrayTranslations;
```

Comment:
```suggestion
  translations: ArrayTranslations;
```
I think this should not be optional as we always have default values to initialize these.
If these are optional, renderers need to consider this and have their own defaults again.
@sdirix Do you have any concerns with adding these as non-optional?